desktop/renderer/index.html

```diff
@@ -7,8 +7,17 @@
     <link rel="stylesheet" href="./fonticon.css" />
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app"><div title="loading..." style="flex: 1;background-color: #f5f5f5"></div>
     <script>
+      function logToElectron() {
+        ['log', 'warn', 'error'].forEach(key => {
+          window.console[`_${key}`] = window.console[key]
+          window.console[key] = function() {
+            require('electron').ipcRenderer.send('console.' + key, Array.prototype.slice.call(arguments))
```

Comment:
yup! that's how electron works. the browser you get has require and access to all electron modules